hw/lint/tools/dvsim/veriblelint.hjson

```diff
@@ -6,10 +6,11 @@
 
   // TODO(#1342): switch over to native structured tool output, once supported by Verible
   // Verible lint-specific results parsing script that is called after running lint
-  report_cmd: "{lint_root}/tools/{tool}/parse-lint-report.py "
+  report_cmd: "{proj_root}/util/dvsim/{tool}-report-parser.py "
   report_opts: ["--repdir={build_dir}",
                 "--outdir={build_dir}"]
-
-  // This customizes the report format for style lint
-  is_style_lint: True
+  overrides:[
+    // This customizes the report format for style lint
+    {name: "title_postfix", value: "Style Lint Results"}
```

Comment:
Clearly, some differentiation is needed for cdc. I would call that a separate flow (note that flow is a `dvsim` keyword). You could extend `CdcCfg` from `LintCfg` that only updates the `flow` variable.

Reply:
I initially had it that way but tried to align with `LintCfg` entirely so that we do not need to add more classes. But if you think it would be cleaner I'll add a `CdcCfg` class for this. The updates in this PR should make it straightforward to extend from `LintCfg`.